scripts/schema_to_enums_v201.py

```diff
@@ -92,15 +91,21 @@ def parse_schema(schema):
         print("Pass path to folder with schemas")
         sys.exit(-1)
 
+    # The second argument of argv is the path for the schemas
     p = Path(sys.argv[1])
     schemas = list(p.glob("*.json"))
 
-    for schema in schemas:
-        parse_schema(schema)
+    # This loop will update the lists enum_types and enum_types_names:
+    # The former contains the enum objects that are later parsed to the
+    # enum.py, using the str representation. And the latter, is used to keep
+    # track of the already processed Enums, avoiding data duplication
+    for schema_ in schemas:
+        parse_schema(schema_)
```

Comment:
Can't you use `schema` over `schema_`?

Reply:
My linter complains because schema shadows the use of schema in parse_schema

Reply:
Isn't that a problem with your linter? The scopes of both variables are totally different. `schema_` lives inside the main, while `schema` lives inside `parse_schema()`

Reply:
You are right